addons/web/static/src/views/fields/percent_pie/percent_pie_field.scss

```diff
@@ -9,6 +9,15 @@
     }
 }
 
+.o_list_renderer .o_list_table .o_list_number{
+    .o_field_percent_pie{
+        display: inline-flex !important;
+    }
+    .o_pie_info .o_pie_text {
+        display: none !important;
```

Comment:
We try to avoid the use of `!important` and this rule is also relatively redundant with the ones below, a preferred fix would be to change the rule that overrides the `display: none` inside [list_renderer.scss](https://github.com/odoo/odoo/blob/1bca81fac0e90289e09dd027a5b2bcae8d851892/addons/web/static/src/views/list/list_renderer.scss#L308), you can add another exception to the rule that overrides the display, something like this:
```scss
span:not(.o_m2o_avatar):not(.o_percent_pie) {
    @include o-text-overflow(inline);
}
```

Reply:
Hello @BastienFafchamps ,

I've made changes. Could you please review them?

Thank you :slightly_smiling_face: